build-system/tasks/report-test-status.js

```diff
@@ -63,13 +60,7 @@ function inferTestType() {
     return `${type}/local-changes`;
   }
 
-  if (IS_SAUCELABS_BETA) {
-    return `${type}/saucelabs-beta`;
-  } else if (IS_SAUCELABS_STABLE) {
-    return `${type}/saucelabs-stable`;
-  } else if (IS_SAUCELABS) {
-    return `${type}/saucelabs`;
-  } else if (IS_DIST) {
+  if (IS_DIST) {
```

Comment:
Nit: Since this is now so simple, we could one-liner it:
```
return `${type}/${IS_DIST ? 'minified' : 'local'}`;
```

Reply:
Combined all three clauses.